src/util.py

```diff
@@ -3,7 +3,6 @@
 
 class OS(Enum):
 
-    LINUX = 'LINUX'
-    MACOS = 'MACOS'
-    WINDOWS = 'WINDOWS'
-
+    LINUX = "LINUX"
+    MACOS = "MACOS"
+    WINDOWS = "WINDOWS"
```

Comment:
Just to check: rre you using `black` for formatting as described in [CONTRIBUTION.md](https://github.com/martomi/chiadog/blob/main/CONTRIBUTING.md#formatting-and-linting)? Let me know if you need help to setup the parts there and if we need to update the manual. I will also have a look how to add some of these steps as automated PR checks soon.

Reply:
Yes, this change was proposed by `black`. All in all that required very little setup.
I did notice that the documentation for running unit tests is not cross-platform. Setting the os environment variables as proposed by the documentation is not possible from a Windows command line, so I'll have to check if there is another (cross-platform) method to get those variables passed into the unit test

Reply:
I was able to set the environment variables on Windows and updated the CONTRIBUTING.md manual to reflect those changes (will send a different PR for this)